src/IO/H5/StellarCollapseEos.hpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::StellarCollapseEos
+
+#pragma once
+
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "DataStructures/BoostMultiArray.hpp"
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ * \brief Reads in tabulated equation of state file from stellarcollapse.org
+ *
+ * Reads in H5 file containing data for tabulated equation of state.
+ * Contains functions to obtain thermodynamic quantities from the file,
+ * stored as either rank-1 or rank-3 datasets.
+ *
+ * It is assumed that the file is in the format of the SRO
```

Comment:
It would be good to provide a list of names of the data that can be read and what it corresponds to physically.

Reply:
That list is provided in the pdf that is linked in the last paragraph

Reply:
Okay, sounds good!